packages/odyssey-react-mui/src/Toast.tsx

```diff
@@ -0,0 +1,155 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import React, {
+  useContext,
+  useEffect,
+  memo,
+  forwardRef,
+  ForwardedRef,
+} from "react";
+import {
+  Alert,
+  AlertTitle,
+  Button,
+  CloseIcon,
+  Link,
+  Snackbar,
+  visuallyHidden,
+} from ".";
+import { ToastStackContext } from "./ToastStackContext";
+
+export type ToastProps = {
+  /**
+   * If set, this determines how long the toast should appear before automatically
+   * disappearing, in milliseconds. It will only take effect if the toast is not dismissable.
+   * If left blank, it defaults to 6000.
```

Comment:
```suggestion
   * If set, this determines how long the toast should appear before automatically disappearing in milliseconds.
   * It will only take effect if the toast is not dismissible.
   * If left blank, it defaults to 6000.
```

Fixed spelling for "dismissible" and unnecessary comma after "disappearing".